packages/litqa/README.md

```diff
@@ -0,0 +1,19 @@
+# aviary.litqa
+
+LitQA environment implemented with aviary,
+allowing agents to perform question answering on the LitQA dataset.
+
+## References
```

Comment:
Can you pout a reference to the LAB-Bench paper, since that's where LitQA2 comes from: https://arxiv.org/abs/2407.10362